### ISSUE TITLE
NullPointerException registerStatement

### ISSUE THREAD

**Issue by mibby** [Phoenix616/ChestShopNotifier#7]
ChestShopNotifier v1.1.5

```
[04:00:29] [Craft Scheduler Thread - 3/WARN]: Exception in thread "Craft Scheduler Thread - 3" 
[04:00:29] [Craft Scheduler Thread - 3/WARN]: org.apache.commons.lang.UnhandledException: Plugin ChestShopNotifier v1.1.5 generated an exception while executing task 150
	at org.bukkit.craftbukkit.v1_9_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
	at com.mysql.jdbc.ConnectionImpl.registerStatement(ConnectionImpl.java:4388)
	at com.mysql.jdbc.StatementImpl.<init>(StatementImpl.java:329)
	at com.mysql.jdbc.ConnectionImpl.createStatement(ConnectionImpl.java:2439)
	at com.mysql.jdbc.ConnectionImpl.createStatement(ConnectionImpl.java:2421)
	at com.wfector.notifier.ChestShopNotifier$2.run(ChestShopNotifier.java:208)
	at org.bukkit.craftbukkit.v1_9_R2.scheduler.CraftTask.run(CraftTask.java:58)
	at org.bukkit.craftbukkit.v1_9_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more
```

```
[16:00:24] [Craft Scheduler Thread - 0/WARN]: Exception in thread "Craft Scheduler Thread - 0" 
[16:00:24] [Craft Scheduler Thread - 0/WARN]: org.apache.commons.lang.UnhandledException: Plugin ChestShopNotifier v1.1.5 generated an exception while executing task 171
	at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
	at com.mysql.jdbc.StatementImpl.<init>(StatementImpl.java:344)
	at com.mysql.jdbc.ConnectionImpl.createStatement(ConnectionImpl.java:2439)
	at com.mysql.jdbc.ConnectionImpl.createStatement(ConnectionImpl.java:2421)
	at com.wfector.notifier.ChestShopNotifier$2.run(ChestShopNotifier.java:208)
	at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftTask.run(CraftTask.java:58)
	at org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
	... 4 more
```

**Comment by mibby** [comment]
Active development @Phoenix616 ? :D  Happy to see new commits!

**Comment by Phoenix616** [comment]
This should also be fixed with 93b9c2ba. If you see it happen again in the future, just reopen this issue.
